### PR TITLE
update README file.

### PR DESCRIPTION
**Pull request by jetakazono**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Northcoders News API is an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
+
+### Enivronment variables
```
Comment (itskatherine) [inline README.md]: Tiny typo here

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
...
+Create two .env files: .env.test and .env.development. Into each, add PGDATABASE=<database_name_here>, with the correct database name for that environment. 
+.env files must be .gitignored
```
Comment (itskatherine) [inline README.md]: Cool, clear instructions 👍 
If you add backticks \` around words you can make them look `all codey`, which could be useful to use here.

**Review by itskatherine — COMMENTED** [review]
I've added some slight improvements Jessica, could you also add an instruction for the user about using `npm i` to get the project set up.
Once you've made these changes, you're ready to merge 💪

File: README.md (hunk 1 above)
Comment (jetakazono) [reply README.md]: will do :)

**Pull request merged**